### PR TITLE
Resolve crashing Sample Details page

### LINKED ISSUE #4676: Sample Details page is crashing
**Describe the bug**
When I open the sample details page, the page just goes blank.

**To Reproduce**

1. Go to https://care.coronasafe.in/sample/6c226570-3e9a-4630-b830-f3bfe138bf9c
2. See error

**Screenshots**
image

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

### PR DESCRIPTION
**Pull request by Pranshu1902**
Fixes #4676 
Resolved crashing sample details page
image

### CODE REVIEW

**Comment by nihal467** [comment]
LGTM

File: src/Components/Patient/SampleDetails.tsx [hunk 1]
```diff
@@ -431,7 +431,7 @@ export const SampleDetails = (props: SampleDetailsProps) => {
               {yesornoBadge(sampleDetails.patient_has_suspected_contact)}
             </div>
             {sampleDetails.patient_travel_history &&
-              sampleDetails.patient_travel_history !== "[]" && (
+              sampleDetails.patient_travel_history.length !== 0 && (
```
Comment (khavinshankar) [inline src/Components/Patient/SampleDetails.tsx:434]: @Pranshu1902 shouldn't it be 

```suggestion
              JSON.parse(sampleDetails.patient_travel_history).length !== 0 && (
```

Comment (Pranshu1902) [reply src/Components/Patient/SampleDetails.tsx:434]: @khavinshankar I think it was correct as before. Getting this error after adding `JSON.parse`
image

Comment (khavinshankar) [reply src/Components/Patient/SampleDetails.tsx:434]: @Pranshu1902 cool, looks good

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**